SignalMessaging/utils/UIUtil.h

```diff
@@ -20,8 +20,6 @@ typedef void (^completionBlock)(void);
 + (void)applyRoundedBorderToImageView:(UIImageView *)imageView;
 + (void)removeRoundedBorderToImageView:(UIImageView *__strong *)imageView;
 
-+ (completionBlock)modalCompletionBlock;
-+ (void)applyDefaultSystemAppearence;
-+ (void)applySignalAppearence;
++ (void)setupSignalAppearence;
```

Comment:
So we _don't_ need to call this in the share extension?

Reply:
Right.